portality/templates/dashboard/index.html

```diff
@@ -14,65 +14,72 @@
             "text" : "<strong>Ready</strong> application",
             "colour" : "var(--dark-green)",
-            "feather": "thumbs-up"
+            "feather": "check-circle",
+            "status": "ready"
         },
         constants.TODO_MANED_COMPLETED: {
             "text" : "<strong>Completed</strong> application",
             "colour" : "var(--mid-green)",
-            "feather": "check-circle"
+            "feather": "user-check",
+            "status": "completed"
         },
         constants.TODO_MANED_ASSIGN_PENDING: {
             "text" : "<strong>Pending</strong> application",
             "colour" : "var(--yellow)",
-            "feather": "inbox"
+            "feather": "inbox",
+            "status": "pending"
         }
     }
 %}
 
 {% block content %}
-<main>
-  <h1>Hi, {{ current_user.name }}!</h1>
-
   <section>
-    <h2 class="type-04">Here are your priorities:</h2>
-    <ol class="unstyled-list todo-list">
+    <h2 class="type-04">
+      You have <strong>{{ todos|length }}{% if todos|length > config.get("TODO_LIST_SIZE")%}+{% endif %}</strong> items in your priority list:
+    </h2>
+    <ol class="todo-list">
       {% for todo in todos %}
       {# TODO only show tasks for this user’s groups #}
       {# TODO integrated priority in list display
       {{ todo.boost }}
        #}
        {% set app_url =  url_for('admin.application', application_id=todo.object_id) %}
       <li
-        class="card card--compact todo-list__item"
+      class="todo-list__item"
         data-app-url="{{ app_url }}"
-        style="border-color: {{ TODOS[todo.action_id[0]].colour }};"
       >
-        <article class="row">
-          <time datetime="{{ todo.date.strftime('%Y-%m-%d') }}" class="col-xs-1 label label--secondary font-mono">
-            <span style="font-size: 27px;">{{ todo.date.strftime('%d') }}</span><br>
-            <span style="font-size: 18.55px;">{{ todo.date.strftime('%b') }}</span><br>
-            <span style="font-size: 14.35px;">{{ todo.date.strftime('%Y') }}</span>
-          </time>
-          <div class="col-xs-11 flex-space-between">
+        <article class="card card--compact" style="border-color: {{ TODOS[todo.action_id[0]].colour }};">
+          <div class="flex-space-between flex-col">
+            <time datetime="{{ todo.date.strftime('%Y-%m-%d') }}" class="label label--secondary font-mono" title="{% if todo.date_type == 'last_manual_update' %}Last updated{% else %}First submitted{% endif %} {{ todo.date.strftime('%A, %-d %B %Y') }}">
+              <span style="font-size: 22px;">{{ todo.date.strftime('%d') }}</span><br>
+              <span style="font-size: 13.55px;">{{ todo.date.strftime('%b') }}</span><br>
+              <span style="font-size: 9.35px;">{{ todo.date.strftime('%Y') }}</span>
+              {# TODO link to search results for this kind of task, across all editor groups #}
+            </time><br/>
+            {% set status = TODOS[todo.action_id[0]].status %}
```

Comment:
I have modified this so that a `link` property is supplied in the `TODO` section instead, and then I've made those links work.  This gives us more freedom to have more complex, non-status related links, as this feature develops

Reply:
Looks good! Thank you.